packages/create-gatsby/src/index.ts

```diff
@@ -0,0 +1,59 @@
+import { prompt } from "enquirer"
+import questions from "./questions.json"
+import { initStarter } from "./init-starter"
+
+interface IAnswers {
+  project: string
+  styling?: string
+  cms?: string
+  features?: Array<string>
+}
+
+export async function run(): Promise<void> {
+  console.log(
+    `Welcome to Gatsby! This command will generate a new Gatsby site for you with the setup you select. Let's answer some questions:`
+  )
+  const data = await prompt<IAnswers>(questions)
+
+  const messages: Array<string> = [
+    `🛠  Create a new Gatsby site called ${data.project}`,
+  ]
+  if (data.cms) {
+    messages.push(`📚 Install and configure the plugin for ${data.cms}`)
+  }
+
+  if (data.styling) {
+    messages.push(
+      `🎨 Get you set up to use ${data.styling} for styling your site`
+    )
+  }
+
+  if (data.features) {
+    messages.push(`🔌 Install ${data.features.join(`, `)}`)
+  }
+
+  console.log(`
+Thanks! Here's what we'll now do:
```

Comment:
This reminds me of what Shannon and Kyle were doing on recipes, maybe we need to have a big get together where we all synthesize ideas.

Reply:
Yeah, this is like a super-simplified, hard-coded version.